### ISSUE TITLE
Rubymine instructions incorrect

### ISSUE THREAD

**Issue by shkm** [samphippen/rubyfmt#344]
Screenshot shows arguments as `-i $FilePath$` but it should be `-i -- $FilePath$`. I'd make a PR but I fear the screenshot would be lower res and therefore uglier.

Alternatively, you could provide an XML file to import the watcher. I exported mine here:

```xml
<TaskOptions>
  <TaskOptions>
    <option name="arguments" value="-i -- $FilePath$" />
    <option name="checkSyntaxErrors" value="true" />
    <option name="description" />
    <option name="exitCodeBehavior" value="ERROR" />
    <option name="fileExtension" value="rb" />
    <option name="immediateSync" value="true" />
    <option name="name" value="rubyfmt" />
    <option name="output" value="$FilePath$" />
    <option name="outputFilters">
      <array />
    </option>
    <option name="outputFromStdout" value="false" />
    <option name="program" value="rubyfmt" />
    <option name="runOnExternalChanges" value="true" />
    <option name="scopeName" value="Project Files" />
    <option name="trackOnlyRoot" value="false" />
    <option name="workingDir" value="" />
    <envs />
  </TaskOptions>
</TaskOptions>
```

Thank you for your work!

**Comment by fables-tales** [comment]
pull requests welcome!

**Comment by reese** [comment]
Fixed in #353, thanks for the update!
